### PR TITLE
feat(esl-footnotes): add back-to-note-label attribute and renderIndex for connected note

### PR DESCRIPTION
**Pull request by dshovchko**
In the scope of PR:
- added separate method to render the content of the connected note
- added back-to-note-label attribute to outputting title for the button

### CODE REVIEW

File: src/modules/esl-footnotes/core/esl-note.ts [hunk 1]
```diff
@@ -46,9 +46,17 @@ export class ESLNote extends ESLBaseElement {
...
     return this._index;
   }
+  public set index(value: number) {
+    this._index = value;
+    this.innerHTML = this.renderedIndex;
+  }
+
+  public get renderedIndex(): string {
+    return `${this._index}`;
+  }
```
Comment (ala-n) [inline src/modules/esl-footnotes/core/esl-note.ts:59]: Is it in purpose to be rewritten?

**Review by ala-n — APPROVED** [review]

File: src/modules/esl-footnotes/core/esl-note.ts (hunk 1 above)
Comment (dshovchko) [reply src/modules/esl-footnotes/core/esl-note.ts:59]: That's right.

Comment (dshovchko) [reply src/modules/esl-footnotes/core/esl-note.ts:59]: And since the rewritten value must be used by the footnote component.

**Review by fshovchko — APPROVED** [review]

**Pull request merged**